### PR TITLE
Profile v2

### PR DESCRIPTION
**Pull request by netanelC**
New 3D profile.

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

**Comment by netanelC** [comment]
Every time we generate the file: "generated.ts", in line 88 will be: 'default: () => 'CURRENT_TIMESTAMP''. So, we need to remember to remove the quote from the start and the end. I didn't manage to solve this in mc-models.

File: db/migration/1659267643836-v2ProfileFinal.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "records" DROP COLUMN "srs_origin"`);
+    await queryRunner.query(`ALTER TABLE "records" DROP COLUMN "production_method"`);
+    await queryRunner.query(`ALTER TABLE "records" DROP COLUMN "nominal_resolution"`);
+    await queryRunner.query(`ALTER TABLE "records" DROP COLUMN "relative_accuracy_le_90"`);
+    await queryRunner.query(`ALTER TABLE "records" ADD "relative_accuracy_se_90" real`);
+    await queryRunner.query(`ALTER TABLE "records" ADD "product_source" text NOT NULL`);
+    await queryRunner.query(`ALTER TABLE "records" ADD "product_status" text DEFAULT 'UNPUBLISHED'`);
+    await queryRunner.query(`ALTER TABLE "records" ALTER COLUMN "update_date" SET DEFAULT now()`);
```
Comment (RonitKissis) [inline db/migration/1659267643836-v2ProfileFinal.ts:14]: why update_date with now and insert with timestamp

File: helm/values-dev.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@ global:
   environment: {}
 image:
   repository: 3d-ingestion-catalog
-  tag: v3.1.0
-enabled: true
+  tag: v4.0.0
+  enabled: true
```
Comment (RonitKissis) [inline helm/values-dev.yaml]: remove annotation- helm is sensitive

**Review by RonitKissis — APPROVED** [review]
Amazing work!!!!!!!!

**Pull request merged**